test/helpers/etherscanInstanceContracts.json

```diff
@@ -1,7 +1,7 @@
 {
   "1": [
     {
-      "address": "0x1f3c21c9d16c5bfc8956a433ec34bfcdeaa9d740",
+      "address": "0x1F3c21c9D16C5Bfc8956a433eC34bfcDEAa9D740",
```

Comment:
Why did you make these checksummed? The endpoint should also accept non-checksummed adresses

Reply:
Because we use them also when checking them at the end. The endpoint used to return non checksummed addresses, I thought it was wrong. But after fixing it, the returned checksummed address needed to be compared with the right value.

We have other tests specifically build to check the checksumming so it's not important to test them also here.